common/test/acceptance/pages/lms/problem.py

```diff
@@ -147,6 +146,27 @@ def is_focus_on_problem_meta(self):
         """
         return self.q(css='.problem-header').focused
 
+    def is_submit_disabled(self):
+        """
+        Checks if the submit button is disabled
+        """
+        return (self.q(css='.problem .submit').attrs('disabled') and
+                'is-disabled' in self.q(css='.problem .submit').attrs('class')[0])
+
+    def is_focus_on_submit_notification(self):
+        """
+        Check for focus submit notification.
+        """
+
+        def focus_check():
+            """
+            Checks whether or not the focus is on the notification-submit
+            """
+            return self.q(css='.notification-submit').focused
+
+        self.wait_for(promise_check_func=focus_check, description='Waiting for the notification-submit to gain focus')
+        return focus_check()
```

Comment:
@staubina 
The return value here is odd. I would rename this method "wait_for_focus_on_submit_notification". If focus is not on the notification, the wait_for will time out, causing a test failure before one would get to the return value. So there is no need to return anything.